modules/signals/spec/with-linked-state.spec.ts

```diff
@@ -0,0 +1,417 @@
+import { TestBed } from '@angular/core/testing';
+import { withMethods, withState } from '../src';
+import { signalStore } from '../src/signal-store';
```

Comment:
unit tests for `withLinkedState` should be defined without `signalStore`. Check `withState` unit tests for inspiration: https://github.com/ngrx/platform/blob/main/modules/signals/spec/with-state.spec.ts#L6

integration tests with `signalStore` can be added to `signal-store.spec.ts`

---

Note regarding imports: all public APIs can be imported from `'../src'`

Reply:
Done